posthog/temporal/delete_recordings/activities.py

```diff
@@ -132,32 +124,18 @@ async def load_recordings_with_person(input: RecordingsWithPersonInput) -> list[
         logger = LOGGER.bind()
         logger.info(f"Loading all sessions for {len(input.distinct_ids)} distinct IDs")
 
-        team = await Team.objects.aget(id=input.team_id)
-
-        logger.info("Building ClickHouse query")
-        query = SessionRecordingListFromQuery(
-            team,
-            RecordingsQuery(distinct_ids=input.distinct_ids, date_from=f"-{team.session_recording_retention_period}"),
-        )
-
-        hogql_query = await database_sync_to_async(query.get_query)()
-
-        hogql_context = HogQLContext(
-            team=team,
-            output_format="JSON",
-            enable_select_queries=True,
-        )
-
-        raw_query = await database_sync_to_async(print_ast)(
-            hogql_query,
-            hogql_context,
-            "clickhouse",
-        )
+        query: str = SessionReplayEvents.get_sessions_from_distinct_id_query(format="JSON")
+        parameters = {
+            "team_id": input.team_id,
+            "distinct_ids": input.distinct_ids,
+            "python_now": datetime.now(pytz.timezone("UTC")),
+            "ttl_days": 365,
+        }
 
-        logger.info("Querying ClickHouse")
+        raw_response: bytes = b""
         async with get_client() as client:
             async with client.aget_query(
-                query=raw_query, query_parameters=hogql_context.values, query_id=str(uuid4())
+                query=query, query_parameters=parameters, query_id=str(uuid4())
```

Comment:
cool cool, so bare CH sql into the CH client instead of hogql

Reply:
Yes - although the previous implementation just dumped the hogQL query into a string anyways so we could use the async CH client from the `temporal.common` module. Also one of the other activities queries CH in exactly the same way with the block-listing-query so with this change at least we're consistent across the activities.

Happy to spend some more time here and implement both queries as HogQL objects if you think that's worth the effort?

Reply:
(and a hoglq perf problem)

but it might also point at something silly that's crept into the replay listing code over time